code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -96,6 +96,73 @@
 	can_charge = TRUE
 	recharge_rate = 1
 
+/obj/item/gun/magic/wand/antag
+	name = "wand of antag"
+	desc = "This wand uses the powers of bullshit to turn anyone it hits into an antag"
+	school = SCHOOL_FORBIDDEN
```

Comment:
If this is for debugging I think it should be locked behind `ifdef TESTING`. Not sure I want this in the ecosystem.